README.md

```diff
@@ -16,69 +16,60 @@ Right now its responsibilities cover:
 
 Example: (Updated)
 
-# Quick start
+# Quick start with Buddy cli
 
-# To run your code locally
-
-Start by forking the example repo https://github.com/ministry-of-silly-code/examples
+## Requirements
 
-```shell
-# 1 - Create a new virtual env
-python3 -m venv venv
-source ./venv/bin/activate
+1. You now need to setup the cluster SSH key. how create and add your SSH keys can be found [here](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/adding-a-new-ssh-key-to-your-github-account) 
+1. Add cluster public-key to: `https://github.com/settings/keys` if you don't know this yet, check this out [here](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/adding-a-new-ssh-key-to-your-github-account)
+1. `pip install wandb && wandb init` there are two ways to set up wandb: 
+    - set up wandb init from the cmd
+    - or set export  WANDB_API_KEY = your wandb key which can be found here: https://wandb.ai/settings
 
-# 2 - Clone [or create] your project, in this case I'm using a basic mnist_classifier 
-git clone https://github.com/[your-username]/examples.git
-cd examples
 
-# 3 - Install Buddy
-# (and add it to you your requirements.txt if you are not using the given examples)
-pip install -e git+https://github.com/ministry-of-silly-code/experiment_buddy.git#egg=experiment_buddy
+## Installation
+### From repo
+```shell
+git clone git@github.com:ministry-of-silly-code/buddy_cli.git
+pip install -e ./buddy-cli
+```
 
-pip install --upgrade -r requirements.txt
+### Directly from pip
+```shell
+pip install git+https://github.com/ministry-of-silly-code/buddy_cli.git@master\#egg\=buddy_cli
+```
 
-# Run your experiments
-python ./mnist_classifier.py
+## Usage
+### Interactive
+```shell
+buddy-init --destination <your_awesome_project>
+cd <your_awesome_project>
 ```
 
-#### Run an experiment remotely
-We need to tell buddy where to deploy the experiments, first add to the file `~/.ssh/config` the following configuration:
+### With options
+```shell
+buddy-init --destination sadafa -y --setup_mila_user <your_awesome_user>
+cd <your_awesome_project>
+```
 
-For Mila users (replace $MILA_USER with your username)
+### Help
 ```shell
-##   Mila
```

Comment:
I would keep the old mode around, so if some one wants to check what is going on and what is written can be done, but just a suggestion.

Reply:
I'm not sure, this is an implementation specific, the important parts in the readme should be how to use init and buddy, everything else is noise